.github/workflows/dart.yml

```diff
@@ -13,11 +13,26 @@ jobs:
         run: |
           git clone --depth=1 https://github.com/flutter/flutter.git
           cd flutter
-          git fetch --depth=1 https://github.com/flutter/flutter.git `cat ../bin/internal/flutter.version`
+          git fetch --depth=1 https://github.com/flutter/flutter.git $(cat ../bin/internal/flutter.version)
           git checkout FETCH_HEAD
       - name: Install pub dependencies
         run: flutter/bin/flutter pub get
       - name: Verify formatting
-        run: flutter/bin/dart format --output=none --set-exit-if-changed lib
+        run: flutter/bin/dart format --output=none --set-exit-if-changed lib test
       - name: Analyze project source
         run: flutter/bin/dart analyze --fatal-infos lib
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch Flutter SDK
+        run: |
+          git clone --depth=1 https://github.com/flutter/flutter.git
+          cd flutter
+          git fetch --depth=1 https://github.com/flutter/flutter.git $(cat ../bin/internal/flutter.version)
+          git checkout FETCH_HEAD
```

Comment:
`actions/checkout` can be used. 

```yaml
      - run: echo "FLUTTER_REV=$(cat ../bin/internal/flutter.version)" >> $GITHUB_ENV
      - uses: actions/checkout@v2
         with:
           repository: flutter/flutter
           ref: ${{ env.FLUTTER_REV }}
           path: flutter
```

Reply:
Will `>> $GITHUB_ENV` work in powershell?